src/classic.js

```diff
@@ -36,8 +32,7 @@ export default class ClassicEditor extends StandardEditor {
 
 		this.data.processor = new HtmlDataProcessor();
 
-		this.ui = new ClassicEditorUI( this );
-		this.ui.view = new BoxedEditorUIView( this.locale );
+		this.ui = new ClassicEditorUIView( this, this.locale );
```

Comment:
Maybe `editor.view` now makes sense. In fact `ClassicEditor` is a controller, and `ClassicEditorUIView` is his view.